src/git/commit_filter.go

```diff
@@ -77,7 +77,7 @@ func ExcludedFiles(excludedFiles ...string) CommitFilterOptionFunc {
 }
 
 func isDirNameInvalid(dir string) bool {
-	return dir == "." || dir == ".." || strings.HasPrefix(dir, "/") || strings.Contains(dir, "./")
+	return dir == "." || dir == ".." || dir == "" || strings.HasPrefix(dir, "/") || strings.Contains(dir, "./")
```

Comment:
Maybe it is too much but (because I fear we are relying on integration tests only), would it make sense to include an additional test to check the new validation? It would have `--included-files=invented,` or something similar.

Reply:
I can add one having `["invented", ""]` and seeing that is refused,